### ISSUE TITLE
Impossible to gain progress for the Mini Chest backpack skin if Lootr is installed

### ISSUE THREAD

**Issue by Not-February** [MrCrayfish/Backpacked#86]
I've opened two buried treasure map chests on a server with Lootr installed and neither time did I gain progress for the skin.

Minecraft 1.18.1
Forge 39.1.2
Backpacked 2.1.1
Lootr 0.1.15.50

**Comment by MrCrayfish** [comment]
Yeah, it won't be compatible since that mod replaces vanilla chests. I can probably add a patch, it's just annoying since I shouldn't have to consider that mod when I am developing a mod.

**Comment by KajesLorian** [comment]
Though more of a work around than a fix, in the Lootr config file you can blacklist buried treasure so you can locate and earn credit for the Mini Chest like so:

loot_structure_blacklist = ["minecraft:buried_treasure"]

This negates one of the uses of Lootr on servers to do this, and I do not recommend keeping this config set like so after all players have earned the achievement. Again this is only a work around and not a fix.
